cmd/serve.go

```diff
@@ -119,9 +119,13 @@ func serve() {
 		DistFS: distFS,
 	}
 
+	wpool := pool.NewWorkerPool(specs.OpenFGAWorkersTotal, tracer, monitor, logger)
```

Comment:
worker pools are used to have control over the amount of resources we use.
The other worker pool still interacts with OFGA. My suggestion is to reuse that one (or delete that one and reuse this) since we always hit the same OpenFGA

Reply:
Even though I think I agree with @shipperizer, this is the same pool that we use for all our other components

Reply:
Thanks, sorry I mistook that